From b275bb75020c653598bf510d2c9b566543666f8a Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Arkadiusz=20Mi=C5=9Bkiewicz?= <arekm@maven.pl>
Date: Mon, 16 Nov 2015 10:52:49 +0100
Subject: [PATCH v2] Xorg.wrap: activate libdrm based detection for KMS drivers
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Xorg.wrap includes code guarded with WITH_LIBDRM for detecting KMS drivers.
Unfortunately it is never activated since code missed to include file
which defines WITH_LIBDRM.

Bugzilla: https://bugs.freedesktop.org/show_bug.cgi?id=92894
Signed-off-by: Arkadiusz Miśkiewicz <arekm@maven.pl>
Reviewed-by: Michel Dänzer <michel.daenzer@amd.com>
Signed-off-by: Hans de Goede <hdegoede@redhat.com>
---
Changes in v2:
-Fix bugzilla reference in commit message
-Add Michel Dänzer's Reviewed-by
---
 hw/xfree86/xorg-wrapper.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/hw/xfree86/xorg-wrapper.c b/hw/xfree86/xorg-wrapper.c
index 4c37cfc..d930962 100644
--- a/hw/xfree86/xorg-wrapper.c
+++ b/hw/xfree86/xorg-wrapper.c
@@ -24,6 +24,7 @@
  */
 
 #include "dix-config.h"
+#include "xorg-config.h"
 
 #include <errno.h>
 #include <fcntl.h>
-- 
2.5.0

